config/locales/en.yml

```diff
@@ -209,6 +209,16 @@ en:
 
 
           The subdomain containing this *FQDN* _must be preconfigured as an Azure DNS zone_. For example, if the *FQDN* is `rancher.example.com`, then `example.com` must be an Azure DNS zone, and Rancher will be served at `https://rancher.example.com` .
+      security:
+        title: "Security options"
+        form_caption: |-
+          The Rancher management server is designed to be secure by default and requires SSL/TLS configuration.
+
+
+          You can choose to use a self-signed TLS certificate provided by the Rancher server, or you may choose to have a [Let's Encrypt](https://letsencrypt.org/) certificate generated for to it.
+
+
+          **NOTE:** *In order for Let's Encrypt to generate a TLS certificate, your Rancher server must be accessible via the Internet at the FQDN provided over TCP port 80 (http), and you must supply an email address for future management.*
```

Comment:
SHouldn't we do this automatically, i.e. if the user chooses letsencrypt we open the port? AS such the masseg would read
```
**NOTE:**  Selecting the Let's Encrypt option will open port 80 in the created network setup and connect the cluster to the Internet. This is required for to generate a Let's Encrypt TLS certificate.
```